### DISCUSSION TITLE
分享「我与隐语」故事，将美好记忆留存，赢取社区好礼🎁

### DISCUSSION THREAD

**Discussion by Candicepan** [secretflow/secretflow#1363] [Announcements]
隐语开源 2 周年啦 🎉 感谢社区小伙伴们的一路陪伴 👏
❤️ 每一位社区小伙伴对隐语社区来说都是最珍贵的，让我们一起把珍贵的记忆/期望与大家分享❤️

分享与隐语的点滴故事，即有机会获取 **隐语社区周边礼品**🎁 

1⃣️ 在隐语官网 **「[🔗 记忆墙](https://www.secretflow.org.cn/zh-CN/events/second-anniversary)」** 或 **本条 Discussion 留言，写下你与隐语的故事，即有机会获得社区周边好礼**
欢迎分享你与隐语的故事呀，也欢迎分享美好照片（参加活动现场合照、收到的周边、获得的证书都欢迎 po 出来～）将在活动结束后选取认真留言的小伙伴送出社区周边🎁哟～
> 文案跟图片都将在「记忆墙」展示哟～上传则默认同意公开展示。

2⃣️ **「[🔗 记忆墙](https://www.secretflow.org.cn/zh-CN/events/second-anniversary)」**：留言内容将在 1 小时后在「[🔗 记忆墙](https://www.secretflow.org.cn/zh-CN/events/second-anniversary)」显示，记得查看你的专属留言噢～～

📏**活动规则**：活动结束后，将选取认真留言的 15 位小伙伴送出社区周边哟～
⏰**活动时间**：2024/7/9 08:00 - 2024/7/22 12:00 (GMT+8)
🎁**活动礼品：**
- 隐语社区周边：定制随行杯 X 5 份
- 隐语社区周边：定制电脑支架 X 5 份
- 隐语社区周边：不锈钢杯子 X 3 份
- 蚂蚁庄园小鸡公仔 X 1 份
- 蚂蚁公仔桌面摆件 X 1 份

 
🎉 **「隐语 2 周年特别线下活动」** 将于 7 月 20 日杭州进行，产学研专家共论道！探索大模型时代数据要素流通突围之道，欢迎[点击报名，锁定名额](https://mp.weixin.qq.com/s/_goeJhfkvnHlBXJeZPpehg)
👉 **隐语代码仓**：欢迎关注隐语的小伙伴点击右上角🌟 Star，第一时间获取开源项目最新动态
🏠 **隐语官网**：[https://www.secretflow.org.cn](https://www.secretflow.org.cn/)

**Comment by jingshi-ant** [comment]
收藏的隐语小贴纸们：
![image](https://github.com/secretflow/secretflow/assets/124752045/f3956f57-6054-421a-9c86-1ab559508950)

**Comment by XiaoHwei** [comment]
之前参与隐语活动时收到的实用充电线，灰常好用！！
![1](https://github.com/secretflow/secretflow/assets/46802602/aba8b022-986a-43ab-bd97-da8d1890e034)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 蚂蚁庄园小鸡公仔 ，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by Quillana** [comment]
密态传输物理设备😏
![IMG_3330 HEIC](https://github.com/secretflow/secretflow/assets/170709029/ea45798e-cfcf-482f-8028-272ff2098f83)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 蚂蚁公仔桌面摆件 ，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by deadlywing** [comment]
SPU里被接收的代码 😄🎉

![image](https://github.com/secretflow/secretflow/assets/25818621/c79c1a70-293f-45b3-8e4e-1eaa2052f809)

**Comment by Yeekin-GYJ** [comment]
第一次线下参与隐语的活动，收获满满！
![image](https://github.com/secretflow/secretflow/assets/101704468/fea502fa-d380-4eab-a770-25f86dde17df)

**Comment by Jamie-Cui** [comment]
贴纸挡大满足！隐语贴纸 yyds

![IMG_1592 HEIC](https://github.com/secretflow/secretflow/assets/32043467/05248027-b70d-4bc8-96ac-f8001ec137d6)

**Comment by Song-Quan** [comment]
在深圳第一次现场参与隐语的线下活动～干货满满收获多多氛围好好！
![img](https://github.com/secretflow/secretflow/assets/171004619/7b204917-8896-412a-882f-ad0c01c59f22)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制电脑支架 ，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by 6fj** [comment]
![image](https://github.com/secretflow/secretflow/assets/99384777/2517a52c-a546-4a14-a880-e22d43a7c3ee)

在西安和大家见面啦～

**Comment by tongke6** [comment]
![image](https://github.com/secretflow/secretflow/assets/124763920/7b684a85-fa13-486c-9ae4-2b20855224ca)

有幸参加北京举办的社区嘉年华活动和大家见面

**Comment by NewByVector** [comment]
有幸见证了隐语的快速发展。
![tmp-1-4Y34F57uSAr3-.JPG](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-4Y34F57uSAr3-.JPG)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制随行杯，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by DuanYuFi** [comment]
首次获得开源社区证书！
![tmp-1-2y6Suh3mClIO-.jpeg](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-2y6Suh3mClIO-.jpeg)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制随行杯，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by DuanYuFi** [comment]
年度必需品
![tmp-1-GCgY8KWtfc5C-.png](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-GCgY8KWtfc5C-.png)

**Comment by DuanYuFi** [comment]
参与隐语线上分享交流～
![tmp-1-xkNf79MoiE83-.png](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-xkNf79MoiE83-.png)

**Comment by DuanYuFi** [comment]
隐语一周年嘉年华～
![tmp-1-i0fmoYZ2VuVt-.png](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-i0fmoYZ2VuVt-.png)

**Comment by Mingbo-Lee** [comment]
从WSL2，到GPU镜像文件，再到多样化的隐语横向联邦学习；从深圳，到杭州，到北京，最后再到厦门；从Contributor，到Activate Contributor，最后再到SFCS；从2022年12月到现在，我很荣幸，也很高兴，能够见证隐语框架和隐语开源社区发展得越来越好！参与隐语开源社区让我体会到合作与分享的力量，结识志同道合的伙伴，共同解决问题。在这里，我不断成长，热爱技术，愿意分享和帮助他人。隐语开源社区让隐私计算技术变得更加开放和进步，非常感谢这宝贵的学习机会！
![mmexport64cb367b7e86ee5d5ce7f199ab2c9dc5_1706103725423](https://github.com/secretflow/secretflow/assets/51162830/ea6cebd6-a835-4e84-8581-44c1cf2ce055)

**Reply by Candicepan** [reply]
♥️ 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制电脑支架，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by haha-zwx-ooo** [comment]
隐语yyds，有幸见证隐语快速发展~!

**Reply by Candicepan** [reply]
♥️ 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制随行杯，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by winnylyc** [comment]
第一次有这么大的舞台让我做分享，非常紧张也非常激动！也非常有幸与各位同样对隐私计算有兴趣的同学及老师交流！
![tmp-1-XUMlNiRJE3k6-.png](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-XUMlNiRJE3k6-.png)

**Reply by Candicepan** [reply]
🎉 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制随行杯，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by Mingbo-Lee** [comment]
第一次参加meetup：杭州meetup !
![mmexport1676767357353](https://github.com/secretflow/secretflow/assets/51162830/14c8f257-b256-43c3-b714-d415b8b03deb)

**Comment by Mingbo-Lee** [comment]
在深圳meetup 的第一次主题分享！
![mmexport1685189440007](https://github.com/secretflow/secretflow/assets/51162830/35d9ea5d-23b0-4e3c-a52d-83f6554acd7d)

**Comment by Mingbo-Lee** [comment]
在隐语 live 进行分享
![屏幕截图 2024-07-08 232704](https://github.com/secretflow/secretflow/assets/51162830/6f53774f-60d5-4d7e-89a1-dcc16b21cbfb)

**Comment by Mingbo-Lee** [comment]
获得社区的表彰~
![image](https://github.com/secretflow/secretflow/assets/51162830/d0b6c362-1d67-4a8b-b53c-5198bcfc4510)

**Comment by Mingbo-Lee** [comment]
获得布道之星！
![image](https://github.com/secretflow/secretflow/assets/51162830/494af2f3-440f-4f1d-a89e-795cb88954d3)

**Comment by Mingbo-Lee** [comment]
在厦门大学参与厦门meetup!
![隐语上海KV（厦门）_画板 1 副本 8-01-01-02](https://github.com/secretflow/secretflow/assets/51162830/16bc74a1-ac6f-4920-bb47-e70635150e1f)

**Comment by Mingbo-Lee** [comment]
主题分享《多样化的隐语联邦学习》
![image](https://github.com/secretflow/secretflow/assets/51162830/9f5058fe-bc04-42f0-a6e9-4f4bc5e6ce9d)

**Comment by YanZhuangz** [comment]
有幸见证了隐语的快速发展~~

**Reply by Candicepan** [reply]
🎉 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制随行杯，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by zoeStartover** [comment]
在过去一年参与了隐语OSCP一期和二期，为隐语社区做了一些微不足道的贡献~ 并且很开心今年能有机会线下在厦门与隐语见面~下面是自己写的并被合并了的代码：
![图片](https://github.com/secretflow/secretflow/assets/69002064/f8a6b5c1-4950-41bd-b727-cad428915db3)

**Reply by Candicepan** [reply]
🥰 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制电脑支架，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by DC-candy** [comment]
很开心参加了隐语二期的活动，也有幸参加了厦门的隐语线下活动，收获良多！隐语是我第一次参加的开源项目，隐语工作人员真的给了我很多帮助，非常感谢隐语活动！
![隐语](https://github.com/secretflow/secretflow/assets/149782865/2a791812-ee20-4d7a-8e52-37ad33e1c9e2)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制电脑支架，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by codedog1229** [comment]
有幸在2024年3月参加了隐语的“隐私计算实训营”，让我对于隐语的强大功能感到惊叹！我们尝试基于隐语平台开发了自己的产品，隐语的开源使得开发者能够快速构建和迭代业务应用，加速数字化转型进程，提升业务创新能力和市场响应速度。希望隐语发展越来越好！！！
![tmp-1-siLmbMfAb80o-.jpg](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-siLmbMfAb80o-.jpg)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：定制电脑支架，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by zinginging** [comment]
隐语YYDS，感谢隐语社区一直以来的支持！
![tmp-1-2W3IjXYCNx8G-.jpg](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-2W3IjXYCNx8G-.jpg)

**Reply by Candicepan** [reply]
😄 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：不锈钢杯子，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by tarantula-leo** [comment]
OSCP的奖品质量非常好！欢迎大家踊跃参加社区共建～
![tmp-1-ORXjURx8ZWwL-.jpg](http://secretflow-community-platform.oss-cn-shanghai.aliyuncs.com/anniversary_pictures/tmp-1-ORXjURx8ZWwL-.jpg)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：不锈钢杯子，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by Zhang-gong** [comment]
基于隐语框架完成了本科毕业设计，收获颇丰，希望未来和隐语一起继续进步！
![image](https://github.com/user-attachments/assets/b45a4cff-16d1-491f-9f18-9b54bb6facb4)

**Reply by Candicepan** [reply]
👏 感谢对隐语的支持～～也期望未来可以继续跟隐语一起探索与贡献更有趣更硬核的数据要素领域新技术～～也恭喜你在本次活动中获得 隐语社区周边：不锈钢杯子，请在 北京时间 7月31日23:59 前，联系 「隐语小助手Calor」（微信号：SecretFlow04 提供邮寄信息～将统一安排奖品寄送哟～

**Comment by hacker-jerry** [comment]
感谢隐语开源如此优雅的隐私计算平台，㊗️隐语越办越好！
